cmd/taskctl/taskctl.go

```diff
@@ -151,6 +151,10 @@ func (tc *TaskCtlCmd) initConfig() (*config.Config, error) {
 	if tc.rootFlags.DryRun {
 		conf.DryRun = tc.rootFlags.DryRun
 	}
+
+	// default set up of summary to true
+	conf.Summary = true
```

Comment:
I thought in the last PR you had it to ! conf.NoSummary?

Reply:
the `no-summary` is a cli flag this is the taskctl internal conf object